src/main/java/org/fundacionjala/at15/katas/pokerhands/diana/Card.java

```diff
@@ -0,0 +1,57 @@
+package org.fundacionjala.at15.katas.pokerhands.diana;
+
+import java.util.ArrayList;
+
+public class Card extends Deck {
```

Comment:
I think that Deck is not a good name for the class because it refers to the complete deck of cards, perhaps a more appropriate name would be PlayerHand, which refers to the cards dealt to a player in the game.